script.js

```diff
@@ -0,0 +1,90 @@
+let humanScore = 0;
+let computerScore = 0;
+
+function getComputerChoice() {
+    const result = Math.floor(Math.random() * 3) + 1;
+
+    if (result === 1)
+        {
+            return "rock";
+        }
+    else if (result === 2)
```

Comment:
From a code formatting perspective, most developers use 
```
if (result == 1) {
    return "rock"
}
```
It's 1 line shorter, and it makes the close } line up properly. This is such a common practice that VSCode will reformat your code. On linux you can use the short-cut Ctrl+Shift+I to reformat the code.